pkg/kube-e2e-test/build.yml

```diff
@@ -0,0 +1,6 @@
+org: linuxkit
+image: kube-e2e-test
+network: true
+disable-content-trust: true
```

Comment:
I setup content trust for this repo so you can now drop this line.

Reply:
Done.